### PR TITLE
[search] Fixed discrepancy in tokens between geocoder and ranker.

### PR DESCRIPTION
**Pull request by ygorshenin**
@mpimenov PTAL

### CODE REVIEW

File: search/search_integration_tests/processor_test.cpp [hunk 1]
```diff
@@ -805,5 +805,33 @@ UNIT_CLASS_TEST(ProcessorTest, SpacesInCategories)
...
+
+  BuildWorld([&](TestMwmBuilder & builder) {
+    builder.Add(country);
+    builder.Add(city);
+  });
+
+  auto id = BuildCountry(country.GetName(), [&](TestMwmBuilder & builder) { builder.Add(street); });
+
+  {
+    auto request = MakeRequest("la France a Paris Rue de la Paix");
```
Comment (mpimenov) [inline search/search_integration_tests/processor_test.cpp]: s/a Paris/à Paris/

File: search/processor.cpp [hunk 2]
```diff
@@ -642,6 +642,46 @@ void Processor::InitParams(QueryParams & params)
   auto & langs = params.GetLangs();
   for (int i = 0; i < LANG_COUNT; ++i)
     langs.insert(GetLanguage(i));
+
+  size_t numStopWords = 0;
```
Comment (mpimenov) [inline search/processor.cpp]: What do you think about factoring this out in a separate function?

File: search/search_integration_tests/processor_test.cpp (hunk 1 above)
Comment (ygorshenin) [inline search/search_integration_tests/processor_test.cpp]: Done

File: search/processor.cpp (hunk 2 above)
Comment (ygorshenin) [inline search/processor.cpp]: Done

**Comment by ygorshenin** [comment]
PTAL

**Comment by ygorshenin** [comment]
PTAL

**Pull request merged**